addons/sys_intercom/XEH_postInitServer.sqf

```diff
@@ -0,0 +1,8 @@
+#include "script_component.hpp"
+
+[QGVAR(infPhoneEventCalling), {[FUNC(infantryPhoneRingingPFH), 1, _this] call CBA_fnc_addPerFrameHandler}] call CBA_fnc_addEventHandler;
+
+["Tank", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Car_F", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Air", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Boat_F", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
```

Comment:
This might be confusing when adding new ones, I'd rather add one for each class on all machines and do stuff based on machine inside the function.